### DISCUSSION TITLE
Help: invoke update

### DISCUSSION THREAD

**Discussion by lucobos** [inventree/InvenTree#2897] [Q&A] [answered]
Hi,
I am trying to make a bare metal install. I think I have followed all steps so far. When executing the command
/home/inventree/src$ invoke update
I get
.....
from cryptography.utils import int_from_bytes
Running InvenTree database migrations...
========================================
2022-04-28 20:38:22,940 ERROR Missing required database configuration value ENGINE
Error: Missing required database configuration value ENGINE

Any ideas about this error?

thanks.
Pedro.

**Comment by matmair** [comment, marked as answer]
Hi there @lucobos
We typically see that error if [this](https://inventree.readthedocs.io/en/latest/start/install/#configure-inventree-options) following step was not done / finished. In particular you have not set the required setting for the database engine, as it is described at the end of the error message. Please configure at least [this section](https://inventree.readthedocs.io/en/latest/start/install/#configure-inventree-options) of your config.yaml - the system will not be able to connect to the database if it is not set correctly.

**Reply by lucobos** [reply]
Thank you @matmair. I had the configuration well set. You helped me to discover my config.yaml was mispelt as config.yalm
Now I'm going to test gunicorn and supervisor
